server/test/5-recipe.route.test.ts

```diff
@@ -1,306 +0,0 @@
-/*tslint:disable:no-unused-expression*/
-
-process.env["NODE_ENV"] = "test";
```

Comment:
would you add new tests into this PR or another?

Reply:
In another I think